.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+* @danipopes @evalir @mattsse
+
+crates/anvil/        @evalir @mattsse
+crates/evm/coverage/ @evalir @onbjerg
+crates/fmt/          @rkrasiuk
+crates/macros/impls/ @danipopes
```

Comment:
wdyt about adding yourself to `.github` too for ci stuff? happy to review there as well

Reply:
Included in the first line